doc/enterprise-shield/c_enterprise_shield_use_cases.md

```diff
@@ -0,0 +1,78 @@
+-   [Home](../../index.md)
+-   [Documentation](../index.md)
+-   Security with KAAZING Gateway
+
+Common Enterprise Shield™ Use Cases ![This feature is available in KAAZING Gateway - Enterprise Edition](../images/enterprise-feature.png)
+=====================================
+
+This topic provides a high-level overview of some common Enterprise Shield™ use cases.
+
+Before You Begin
+--------------------------------------
+
+This procedure is part of [Configure Enterprise Shield™ for KAAZING Gateway](o_enterprise_shield_checklist.md):
+
+1.  [Common KAAZING Gateway Production Topologies](../admin-reference/c_topologies.md#enterprise-shield--)
+2.  [About Enterprise Shield™](o_enterprise_shield_checklist.md#about-enterprise-shield)
+3.  **Common Enterprise Shield™ Use Cases**
+4.  [Setting Up the Gateway](../about/setup-guide.md)
+5.  [Walkthrough: Configure Enterprise Shield™](p_enterprise_shield_config.md)
+6.  [Walkthrough: Configure Enterprise Shield™ for High Availability](p_enterprise_shield_cluster.md)
+
+The use case configurations described in this topic vary depending on the level of security, such as if the Gateways are clustered for high availability, whether network transports are used, and so on. Also, the sample topologies use two Gateway services instead of just one to demonstrate the Enterprise Shield™ reverse connectivity behavior in a configuration with multiple services.
+
+Each of the Enterprise Shield™ use cases configures Enterprise Shield™ on a single pair of Gateways (DMZ and internal Gateways), and then extends that topology to a cluster configuration. To configure Enterprise Shield™ and control Gateway operation, you configure elements in the Gateway configuration (gateway-config.xml) file. For complete information about Gateway configuration, see [About Gateway Configuration](../admin-reference/c_configure_gateway_concepts.md). Also, the complete Gateway configuration files used for these use cases are available on `kaazing.org`. The configuration file names are included in each use case description that follows.
+
+Use Case 1: Test Drive Enterprise Shield™
+------------------------------------------------------------------
+
+This is the simplest Enterprise Shield™ architecture for the purpose of prototyping and evaluating how this feature can meet your business needs. No security is configured, which reduces the complexity of setting up this architecture. Also, it is easy to demonstrate this configuration on a single host environment, such as a laptop.
+
+![Use Case 1: Test Drive Enterprise Shield™](../images/es_usecase3.png)
+
+**Figure 1: Test Drive Enterprise Shield™**
+
+The following complete configuration files are available on `kaazing.org`:
+
+-   Single pair of Gateways: 
+  -   Internal Gateway: `enterprise-shield-use-case-1-single-internal-gw.xml`
```

Comment:
These are the kinds of things I meant earlier that we should track, in terms of links to kaazing.org. Once the content is in place on kaazing.org, these can be updated with real links.